src/components/apiComponents/endPointSelector.jsx

```diff
@@ -0,0 +1,94 @@
+import { Select } from '@headlessui/react'
+import { ChevronDownIcon } from '@heroicons/react/20/solid'
+import clsx from 'clsx'
+import { act, useState } from 'react'
+
+const endPoints = {
+  eclipseAuraMainnet: {
+    name: 'Eclipse Mainnet',
+    uri: 'https://aura-eclipse-mainnet.metaplex.com',
+    value: 'eclipseAuraMainnet',
+  },
+  custom: {
+    name: 'Custom',
+    uri: 'custom',
+    value: 'custom',
+  },
+}
```

Comment:
Assuming we are fine with showing the URLs, right now when you select 'Custom', you get the box showing 'https://aura-mainnet.metaplex.com' that you can modify.  But when you select 'Eclipse Mainnet', it just shows the selector box (as this is a fixed URL).  It would be nicer if it showed the Eclipse Mainnet URL but greyed out.  So that you could see what its actually using.

Reply:
I like that!